packages/anon-aadhaar-contracts/test/Verifier.ts

```diff
@@ -4,23 +4,21 @@ import { ethers } from 'hardhat'
 import { genData, exportCallDataGroth16 } from './utils'
 import { ArgumentTypeName } from '@pcd/pcd-types'
 import {
-  IdentityPCDArgs,
+  AnonAadhaarPCDArgs,
   prove,
   init,
   PCDInitArgs,
   splitToWords,
 } from 'anon-aadhaar-pcd'
+import { WASM_URL, ZKEY_URL, VK_URL } from '../../common/constants'
```

Comment:
Can we see `common` folder as  packages?

Reply:
What do you mean by that? Make common a Node package?

Reply:
Yes, or we move it to anon aadhaar pcd package. I think we should move to pcd package.

Reply:
Moved constants to pcd package 👌